### PR TITLE
make backstories congruous again

### PR DESCRIPTION
**Pull request by j-gynn**
## About The Pull Request

Should let the game work by accessing a private function of `NewCatFactory`. I'm sorry, but this was the most sensible way. In fairness, this is a frustrating edge-case where it's needed outside of new cat.

### CODE REVIEW

**Review by scribblecrumb — APPROVED** [review]

**Review by scribblecrumb — CHANGES_REQUESTED** [review]
nvm your tests are angry

File: scripts/cat/factories/new_cat_factory.py [hunk 1]
```diff
@@ -261,6 +261,7 @@ def _get_random_gender_and_genderalign(cls, age) -> dict:
         return gender
 
     @staticmethod
+    @abstractmethod
```
Comment (rustykitty) [inline scripts/cat/factories/new_cat_factory.py]: Looks like the tests are complaining because of this.

**Comment by j-gynn** [comment]
i hate my life.

**Review by scribblecrumb — APPROVED** [review]
